fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java

```diff
@@ -506,13 +511,44 @@ public Response createObject(@HeaderParam("Content-Disposition") final ContentDi
             session.save();
         } catch (final RepositoryException e) {
             throw new RepositoryRuntimeException(e);
+        } catch (final Exception e) {
```

Comment:
The IOException is the root cause but the exception that bubbles up is actually a BinaryStoreException which is not a subclass of IOException.